package.json

```diff
@@ -1,9 +1,9 @@
 {
   "name": "graphql-2-json-schema",
-  "version": "0.10.0",
+  "version": "0.10.1",
   "main": "dist/index.js",
-  "repository": "git@github.com:wittydeveloper/graphql-to-json-schema.git",
-  "author": "Charly POLY <cpoly55@gmail.com>",
+  "repository": "git@github.com:watermelonexpress/graphql-to-json-schema.git",
+  "author": "BenchPrep Developers <developers@benchprep.com>",
```

Comment:
Updating to watermelonexpress given we've drifted from the fork origin. Open to reverting if we want to keep the original info.